### DISCUSSION TITLE
PLEASE HELP ME ERROR IN PHP.INI CURL ERROR PLEASE HELP ME FIX IT

### DISCUSSION THREAD

**Discussion by ilyxandi** [leokhoa/laragon#615] [Q&A] [answered]
![Screenshot 2023-11-23 001531](https://github.com/leokhoa/laragon/assets/114811436/d3139a9f-55b5-40ad-829d-a9339ba123ba)

**Comment by dustinatx** [comment]
I had this happen. Updating Apache to the latest version fixed it for me.

https://www.kreaweb.be/laragon-update-apache/

**Reply by ilyxandi** [reply]
same man i update apache v17 same please fix it

**Reply by dustinatx** [reply]
Sorry, I don't know what to tell you. This is what I am using:

php-8.2.12-Win32-vs16-x64
httpd-2.4.58-win64-VS17
mysql-8.0.34-winx64

**Reply by hventura** [reply]
Upgrading the Apache to 2.4.58 worked for me.
Thank you for solution 💪

**Reply by isaacdarcilla** [reply]
This works.

**Reply by b0rd1** [reply]
it works for me

**Reply by MeeLight** [reply]
@dustinatx thnks! 👍🏻 It worked for me when I downloaded a newer version of Apache

**Reply by Awais-Koder** [reply]
Thanks man you saved me.

**Comment by ilyxandi** [comment]
[23-Nov-2023 05:55:19 UTC] PHP Warning:  PHP Startup: Unable to load dynamic library 'curl' (tried: D:/Laragon/laragon/bin/php/php-8.3.0-Win32-vs16-x64/ext\curl (The specified module could not be found), D:/Laragon/laragon/bin/php/php-8.3.0-Win32-vs16-x64/ext\php_curl.dll (The specified procedure could not be found)) in Unknown on line 0

**Comment by leokhoa** [comment]
@ilyxandi : Is your Windows OS 32-bit?
If not, can you show a picture that you updated to a different version of Apache?

**Reply by dustinatx** [reply]
@leokhoa  In case it helps... I was using Windows 11 with:

httpd-2.4.54-win64-VS16
php-8.2.10-Win32-vs16-x64

When I upgraded to:

php-8.2.12-Win32-vs16-x64

... I got the same error as @ilyxandi (if I recall correctly). When I updated Apache to:

httpd-2.4.58-win64-VS17

The error went away.

I haven't tried php-8.3.0

**Reply by ilyxandi** [reply]
i solve this error with upgrade php to 8.3 in error, I'm fresh install laragon reinstall laragon with PHP 8.3 and install apache v 17 node 18

**Comment by kierantaylorio** [comment, marked as answer]
I was able to replicate this issue on Windows 11 with a clean install of the latest version of Laragon. 

The issue is present with PHP versions 8.2.X and 8.3.

Upgrading Apache manually to 2.4.58 resolved this issue for me. 

Thank you for your continued work on this project @ Laragon team

**Reply by ilyxandi** [reply]
yes i solve with update php 3.0 and get fresh install laragon again thens for solve men 🙏🦾🦾🦾🤩🤩🤩🤩🤩🤩🤩

**Reply by ParagiDev** [reply]
Yeah... it's work

**Reply by cpajuelodeveloper** [reply]
updated php and apache and works!
